### ISSUE TITLE
out_kafka escaping backslashes unnecessarily

### ISSUE THREAD

**Issue by brunoyb** [fluent/fluent-bit#673]
Hi, I was testing Fluent Bit and found that, apparently, out_kafka is doubling all backslashes from my messages. I'm not sure if I'm doing something wrong or if it really is a problem with the plugin.

Here is what I found so far: using image `fluent/fluent-bit:0.13.4` in a Kubernetes cluster as a sidecar for an application.

Configuration:

```
[SERVICE]
    Flush         1
    Log_Level     info
    Daemon        off
    Parsers_File  parsers.conf

[INPUT]
    Name              tail
    Tag               kube.*
    Path              /logs/access.log
    Parser            json
    DB                /var/log/flb_kube.db
    Mem_Buf_Limit     5MB
    Skip_Long_Lines   On
    Refresh_Interval  10

[OUTPUT]
    Name           kafka
    Match          *
    Brokers        kafka-broker:9092
    Topics         fluent-bit
    Retry_Limit    false
    # hides errors "Receive failed: Disconnected" when kafka kills idle connections
    rdkafka.log.connection.close false
    # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
    rdkafka.queue.buffering.max.kbytes 10240
    # for logs you'll probably want this ot be 0 or 1, not more
    rdkafka.request.required.acks 1
```

When using `kafka-console-consumer.sh` (provided by Kafka) to consume messages from the topic, I can see messages arriving like this:

```
{"@timestamp":1530749644.003532, "date":"2018-07-04 21:13:55", "request":{"method":"GET", "uri":"\\/", ...
```

Notice that the `uri` field has two backslashes.

If I change `OUTPUT` to `stdout` I can see the messages like this:

```
[1530749820.007748000, {"date"=>"2018-07-04 21:16:59", "request"=>{"method"=>"GET", "uri"=>"\/", ...
```

Which is the correct format, `uri` with only one backslash (like the original messages from the `INPUT`).

Am I messing something up?

Thanks in advance!

**Comment by edsiper** [comment]
Please check the following comment on #1278 : 

https://github.com/fluent/fluent-bit/issues/1278#issuecomment-499583503

**Comment by brunoyb** [comment]
Thank you for looking into this! Unfortunately I'm not able to test it right now. It's look good though!

**Comment by edsiper** [comment]
Issue already fixed, ref: #1278 (comment)
